custom-pages/universal-login/src/components/Register/Register.js

```diff
@@ -351,7 +351,7 @@ class Register extends Component {
                 error={errors.tAndC}
                 aria-describedby="tandc-error"
                 value="agree"
-                hint=<a href="https://www.nice.org.uk/terms-and-conditions">Terms and conditions</a>
+                hint=<a href="https://www.nice.org.uk/terms-and-conditions" target="_blank" rel="noopener noreferrer">Terms and conditions <span class="screen-reader-only">(opens in a new tab)</span></a>
```

Comment:
i think you might be able to use the visually-hidden class for this from the design system:

https://nice-digital.github.io/nice-design-system/foundations/visibility/#hide-visually